src/contracts/signer-identity.spec.ts

```diff
@@ -362,91 +332,80 @@ describe('signer-identity (identity based signer)', function test() {
       it('should perform fund transfer to contract with a fallback function, e.g. identities',
         async () => {
           const amountToSend = Math.floor(Math.random() * 1e3);
-          const balanceBefore = new BigNumber(await web3.eth.getBalance(signer.activeIdentity));
-          await executor.executeSend({
-            from: signer.underlyingAccount,
-            to: signer.activeIdentity,
+          const balanceBefore = new BigNumber(await web3.eth.getBalance(account0));
+          await runtimes[0].executor.executeSend({
+            from: account0,
+            to: identity0,
             gas: 100e3,
             value: amountToSend,
           });
-          const balanceAfter = new BigNumber(await web3.eth.getBalance(signer.activeIdentity));
+          const balanceAfter = new BigNumber(await web3.eth.getBalance(account0));
           const diff = balanceAfter.minus(balanceBefore);
           expect(diff.eq(new BigNumber(amountToSend))).to.be.true;
         });
 
       it('should reject fund transfer to new contract and funds should stay with identity ',
         async () => {
           const randomString = Math.floor(Math.random() * 1e12).toString(36);
-          const contract = await executor.createContract(
-            'TestContract', [randomString], { from: signer.activeIdentity, gas: 1e6 },
+          const contract = await runtimes[0].executor.createContract(
+            'TestContract', [randomString], { from: identity0, gas: 1e6 },
           );
           expect(contract).to.be.a('Object');
 
           const amountToSend = Math.floor(Math.random() * 1e3);
-          const balanceBefore = new BigNumber(await web3.eth.getBalance(signer.activeIdentity));
-          await expect(executor.executeSend({
-            from: signer.activeIdentity,
+          const balanceBefore = new BigNumber(await web3.eth.getBalance(account0));
```

Comment:
```suggestion
          const balanceBefore = new BigNumber(await web3.eth.getBalance(identity0));
```

should this be identity?

Reply:
ignore it